prow/plugins/bugzilla/bugzilla.go

```diff
@@ -377,6 +446,20 @@ To reference a bug, add 'Bug XXX:' to the title of this pull request and request
 					response += " The bug has been updated to refer to the pull request using the external bug tracker."
 				}
 			}
+			// if bug is valid and qa command was used, identify qa contact via email
+			if e.command == qaCommand {
+				query := &emailToLoginQuery{}
+				email := bug.QAContactDetail.Email
+				queryVars := map[string]interface{}{
+					"email": githubql.String(email),
+				}
+				err := gc.Query(context.Background(), query, queryVars)
+				if err != nil {
+					response += fmt.Sprintf("\nERROR: Failed to run github query to identify user with email == %s", email)
```

Comment:
The user does not need to know that we tried to run a query -- it's immaterial to them. Can we follow the error formatting for other types of errors that tells users to contact an admin?